packages/manager/apps/zimbra/src/pages/dashboard/emailAccounts/order/Order.page.tsx

```diff
@@ -170,153 +112,155 @@ const OrderCatalogForm = ({
-                value
-                  ? value * starterPlan?.monthly.price
-                  : starterPlan?.monthly.price
-              }
-              tax={
-                value
-                  ? value * starterPlan?.monthly.tax
-                  : starterPlan?.monthly.tax
-              }
-              intervalUnit={starterPlan?.monthly.intervalUnit}
-              ovhSubsidiary={catalog.locale.subsidiary}
-              locale={locale}
-            ></Price>
-          </OdsFormField>
-        )}
-      />
-      <Controller
-        control={control}
-        name="commitment"
-        render={({ field: { name, value, onChange } }) => (
-          <OdsFormField error={errors?.[name]?.message}>
-            <label htmlFor={name} slot="label">
-              <OdsText preset={ODS_TEXT_PRESET.heading4}>
-                {t('zimbra_account_order_subtitle_commitment')}
-              </OdsText>
-            </label>
-            <div className="flex flex-col gap-4">
-              <div className="flex leading-none gap-4">
-                <OdsRadio
-                  id="1-month"
-                  name="1-month"
-                  data-testid="radio-1-month"
-                  value="1"
-                  isChecked={value === '1'}
-                  onClick={() => onChange('1')}
-                ></OdsRadio>
-                <label htmlFor="1-month" className="cursor-pointer">
-                  <OdsText preset={ODS_TEXT_PRESET.paragraph}>
-                    {`1 ${t('zimbra_account_order_commitment_month')}`}
-                  </OdsText>
-                </label>
-              </div>
-              <div className="flex leading-none gap-4">
-                <OdsRadio
-                  id="12-month"
-                  name="12-month"
-                  data-testid="radio-12-month"
-                  value="12"
-                  isDisabled
-                  isChecked={value === '12'}
-                  onClick={() => onChange('12')}
-                ></OdsRadio>
-                <label
-                  htmlFor="12-month"
-                  className="flex flex-col cursor-pointer"
+            <Controller
+              control={control}
+              name={ZimbraPlanCodes.ZIMBRA_STARTER}
+              render={({ field: { name, value, onChange, onBlur } }) => (
+                <OdsFormField
+                  className="flex flex-col gap-4"
+                  error={errors?.[name]?.message}
                 >
-                  <OdsText preset={ODS_TEXT_PRESET.paragraph}>
-                    {`12 ${t('zimbra_account_order_commitment_months')}`}
-                  </OdsText>
-                  <OdsText preset={ODS_TEXT_PRESET.caption}>
-                    {t('zimbra_account_order_commitment_available_soon')}
+                  <OdsText
+                    className="font-bold"
+                    preset={ODS_TEXT_PRESET.paragraph}
+                  >
+                    <b>{starterPlan?.blobs?.commercial?.name}</b>
```

Comment:
```suggestion
                    className="font-bold"
                    preset={ODS_TEXT_PRESET.paragraph}
                  >
                    {starterPlan?.blobs?.commercial?.name}
```
If the classeName is bold i don't think you need <b>